tpl/template.go

```diff
@@ -125,9 +126,20 @@ func (t *TemplateAdapter) Execute(w io.Writer, data interface{}) (execErr error)
 		defer t.Metrics.MeasureSince(t.Name(), time.Now())
 	}
 
+	templatePath := ""
+	if IncludeTemplateRef {
+		if !strings.HasPrefix(t.Name(), "_internal") && t.Name() != "alias" {
```

Comment:
If I enter the block when template name is `alias`, then `fileAndFilename` panics on the stat line. Any help appreciated.